eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -210,18 +212,33 @@ stages:
                   - template: tools/gpg/gpg.yml@azure-sdk-build-tools
                   - ${{ each artifact in parameters.Artifacts }}:
                     - ${{if ne(artifact.skipPublishPackage, 'true')}}:
-                      - template: /eng/pipelines/templates/steps/java-publishing.yml
-                        parameters:
-                          ArtifactID: ${{artifact.name}}
-                          GroupID: ${{artifact.groupId}}
-                          ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
-                          Target: MavenCentral
-                          BuildToolsPath: $(Pipeline.Workspace)/azure-sdk-build-tools
-                          JavaRepoRoot: $(Pipeline.Workspace)/azure-sdk-for-java
-                      - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
-                        parameters:
-                          ArtifactName: ${{parameters.ArtifactName}}-${{artifact.name}}-mavencentral-$(System.JobAttempt)
-                          ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+                      - ${{ if eq(parameters.UseEsrpRelease, 'true') }}:
+                        - template: /eng/pipelines/templates/steps/java-publishing.yml
+                          parameters:
+                            ArtifactID: ${{artifact.name}}
+                            GroupID: ${{artifact.groupId}}
+                            ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+                            OutputDirectory: $(Pipeline.Workspace)/EsrpPackages
+                            Target: EsrpRelease
```

Comment:
Looking to see what the difference is between these two blocks it seems like we could just default things like the output and artifact name based on the Target and simply have one call to the java-publishing.yml while setting the Target via a variable we define above.